server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMailboxesMethodTest.java

```diff
@@ -823,6 +829,68 @@ public void getMailboxesShouldReturnMaxMessageQuotasForInboxWhenSet() throws Exc
             .body(FIRST_MAILBOX + ".quotas['#private&alice@domain.tld']['MESSAGE'].max", equalTo(43));
     }
 
+    @Test
+    @Ignore("Not handled by DefaultQuotaRootResolver for now")
```

Comment:
You can easily share a mailbox from another user to notice QuotaRoot and quota differences....

IMO  THAT would be a valuable test ;-)

Reply:
true, and done